test/suite/wttest.py

```diff
@@ -1209,19 +1209,6 @@ def sanitized_shortid(self):
     def className(self):
         return self.__class__.__name__
 
-def zstdtest(description):
```

Comment:
Is this change just opportunistic? In that nothing uses the condition?

Reply:
The test `test/suite/test_compress02.py` used to use it but it is no longer needed thanks to the following lines:

```
        extlist.skip_if_missing = True
        extlist.extension('compressors', 'zstd')
```
Since it was the only place where it was used and I removed it, I removed the decorator definition entirely.